apps/sms/services/js/moz_settings/moz_settings_shim.js

```diff
@@ -0,0 +1,59 @@
+/* global BridgeServiceMixin */
+
+/* exported MozSettingsShim */
+
+(function(exports) {
+'use strict';
+
+/**
+ * Name of the service for mozSettings API shim.
+ * @type {string}
+ */
+const SERVICE_NAME = 'moz-settings-shim';
+
+/**
+ * Array of method names that need to be exposed for API shim.
+ * @type {Array.<string>}
+ */
+const METHODS = Object.freeze(['get', 'set']);
+
+var mozSettings = null;
+
+var MozSettingsShim = {
+  init(settings) {
```

Comment:
I was thinking about smth like this: https://github.com/mozilla-b2g/gaia/pull/31051/files#diff-0d83de4eba65c6e6dbdf3183d4fcc9f0R6